src/gui/widgets/LcdWidget.cpp

```diff
@@ -235,16 +244,18 @@ void LcdWidget::setMarginWidth( int width )
 
 void LcdWidget::updateSize()
 {
-	int margin = 1;
+	int margin_x1 = m_seamlessLeft ? 0 : 1 + m_marginWidth;
+	int margin_x2 = m_seamlessRight ? 0 : 1 + m_marginWidth;
+	int margin_y = 1;
 	if (m_label.isEmpty()) {
-		setFixedSize( m_cellWidth * m_numDigits + 2*(margin+m_marginWidth),
-				m_cellHeight + (2*margin) );
+		setFixedSize(m_cellWidth * m_numDigits + margin_x1 + margin_x2,
+				m_cellHeight + (2 * margin_y));
 	}
 	else {
 		setFixedSize(qMax<int>(
-				m_cellWidth * m_numDigits + 2*(margin+m_marginWidth),
+				m_cellWidth * m_numDigits + margin_x1 + margin_x2,
```

Comment:
Can this entire statement be cleaned up a little? The tab alignment needs to be fixed. Figure while you were touching it we can fix it.